src/protocol.cpp

```diff
@@ -2174,40 +1996,29 @@ void CProtocol::CreateCLServerListMes ( const CHostAddress&        InetAddr,
         const QByteArray strUTF8City  = vecServerInfo[i].strCity.toUtf8();
 
         // size of current list entry
```

Comment:
And this one.

With all these protocol ones, using `//` single line comments (with just the commented value on the line, of course) might be another approach.

Reply:
@pljones I can change them all to look like this:
```cpp
    // size of current message body
    const int iEntrLen = 2 +                           // server internal port number
                         2 +                           // country
                         1 +                           // maximum number of connected clients
                         1 +                           // is permanent flag
                         2 + strUTF8Name.size() +      // name utf-8 string size
                         2 + strUTF8LInetAddr.size() + // server internal address utf-8 string size
                         2 + strUTF8City.size();       // city utf-8 string size
```

or second option (+ operator on next line) - but that doesn't align that nicely:
```cpp
    // size of current message body
    const int iEntrLen = 2                             // server internal port number
                         + 2                           // country
                         + 1                           // maximum number of connected clients
                         + 1                           // is permanent flag
                         + 2 + strUTF8Name.size()      // name utf-8 string size
                         + 2 + strUTF8LInetAddr.size() // server internal address utf-8 string size
                         + 2 + strUTF8City.size();     // city utf-8 string size
```

Reply:
I have applied the first option as the style definition also prefers breaking after operands